app/models/paper.rb

```diff
@@ -55,4 +55,20 @@ def valid_methodologies
       'case report'
     ]
   end
+
+  # Skips validations, fallthrough to autosave methods for easy import
+  def validate_associated_records_for_authors; true; end
+  def validate_associated_records_for_links; true; end
```

Comment:
for the other branch I'm working on, I'm letting people add links, this might conflict with this validation, but I'll see if there is any issues post merge.